resources/terragrunt_installer.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+include AtlantisCookbook::Helpers
+
+resource_name :terragrunt_installer
+provides :terragrunt_installer
+
+property :checksum, String, regex: /^[a-zA-Z0-9]{64}$/
+property :download_base_url, String, default: 'https://github.com/gruntwork-io/terragrunt/releases/download'
+
+property :group, [String, Integer], default: 'atlantis'
+property :owner, [String, Integer], default: 'atlantis'
+property :mode, [String, Integer], default: '0755'
+
+property :version, String, required: true
```

Comment:
Would it make sense to set a default value for this and the checksum above so it works "out of the box"? I realize this means you may need to update it regularly as upstream has new releases, but at least it'll allow users to do `terragrunt_installer 'default'` for simple stuff and override as they need to.

Reply:
great idea, ill do the same for terraform and atlantis and update the docs